javascript/pokemonAlt.js

```diff
@@ -0,0 +1,19 @@
+$(document).on('click', '#btnMostrar', function(e) {
+  detalles();
+});
+
+function detalles() {
+  var settings = {
+    "async": true,
+    "crossDomain": true,
+    "url": "http://pokeapi.co/api/v2/pokemon/gengar/",
```

Comment:
Noté que todas las veces, mandamos llamar la información del ejemplo de `gengar`. Eso quiere decir que incluso si en la UI pidiéramos el nombre del pokemon a buscar, nuestro request no está preparado para hacer llamados dinámicos. 
Se debe cambiar este request a poder recibir el nombre de un pokemon y mandar a pedir su información a la API, no solo de gengar.